examples/mxnet_imagenet_resnet50.py

```diff
@@ -322,7 +324,8 @@ def evaluate(epoch):
     opt = mx.optimizer.create('sgd', **optimizer_params)
 
     # Horovod: create DistributedTrainer, a subclass of gluon.Trainer
-    trainer = hvd.DistributedTrainer(params, opt)
+    trainer = hvd.DistributedTrainer(params, opt,
```

Comment:
Regarding use in examples: I think we should try to balance presenting the available features to users with not overwhelming them with lots of parameters.  So for some examples (where we only use the defaults) I would suggest keeping them that way, while for the examples that show things like gradient compression and Adasum, I would add this parameter.  Does that sounds reasonable to you?

Reply:
I went ahead and remove the `gradient_predivide_factor` argument from some examples to simplify things.